components/admin_console/openid_convert/openid_convert.tsx

```diff
@@ -0,0 +1,141 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+
+import {FormattedMessage} from 'react-intl';
+
+import {ActionFunc} from 'mattermost-redux/types/actions';
+
+import {AdminConfig} from 'mattermost-redux/types/config';
+
+import {BaseProps} from 'components/admin_console/admin_settings';
+
+import {browserHistory} from 'utils/browser_history';
+
+import FormError from 'components/form_error';
+
+import imagePath from 'images/openid-convert/emoticon-outline.svg';
+
+import './openid_convert.scss';
+
+const openidScope = 'profile openid email';
```

Comment:
Lets store this as a constant globally so it can be used in the admin definition as well.